vmm-reference-simple-handler.md

```diff
@@ -0,0 +1,137 @@
+# [Simple Handler](./../src/devices/src/virtio/net/simple_handler.rs)
+
+`SimpleHandler` contains methods for handling Tx/Rx queues. This handler is generalized over the implementaton of queues (does not make any assumptions the way queues are implemented). 
+
+```rs
+pub struct SimpleHandler<M: GuestAddressSpace, S: SignalUsedQueue> {
+    pub driver_notify: S,
+    pub rxq: Queue<M>,
+    pub rxbuf_current: usize,
+    pub rxbuf: [u8; MAX_BUFFER_SIZE],
+    pub txq: Queue<M>,
+    pub txbuf: [u8; MAX_BUFFER_SIZE],
+    pub tap: Tap,
+}
+
+```
+
+__A brief overview of all the struct variables:__
+
+1. `driver_notify:`
+    
+    A SignalUsedQueue object to model the operation of signalling the driver about used events. It should implement a method named `signal_used_queue` that takes parameter `index` and notifies the driver about index. Index values used here in SimpleHandler are `RXQ_INDEX` (for receiving) and `TXQ_INDEX` (for transmitting).
```

Comment:
A SignalUsedQueue object to model the operation of signaling the driver about used events. It should implement a method named `signal_used_queue` that takes a parameter `index` and notifies the driver about index. The index values used here in SimpleHandler are `RXQ_INDEX` (for receiving) and `TXQ_INDEX` (for transmitting).

Reply:
When giving review comments, you should highlight what action(s) you want the PR author to take. Another way to give this comment could be:

signalling -> signaling
parameter -> a parameter